mmv1/products/privateca/api.yaml

```diff
@@ -419,281 +564,325 @@ objects:
                     description: |
                       Corresponds to OID 1.3.6.1.5.5.7.3.8. Officially described as "Binding the hash of an object to a time".
                   - !ruby/object:Api::Type::Boolean
                     name: 'ocspSigning'
+                    output: true
                     description: |
                       Corresponds to OID 1.3.6.1.5.5.7.3.9. Officially described as "Signing OCSP responses".
                 - !ruby/object:Api::Type::Array
                   name: 'unknownExtendedKeyUsages'
-                  required: true
+                  output: true
                   description: |
                     An ObjectId specifies an object identifier (OID). These provide context and describe types in ASN.1 messages.
                   item_type: !ruby/object:Api::Type::NestedObject
                     properties:
                     - !ruby/object:Api::Type::NestedObject
-                      name: 'obectId'
-                      required: true
+                      name: 'obectId' 
+                      output: true
                       description: |
                         Required. Describes how some of the technical fields in a certificate should be populated.
                       properties:
                       - !ruby/object:Api::Type::Array
                         name: 'objectIdPath'
-                        required: true
+                        output: true
                         item_type: Api::Type::Integer
                         description: |
                           An ObjectId specifies an object identifier (OID). These provide context and describe types in ASN.1 messages.
           - !ruby/object:Api::Type::NestedObject
             name: 'publicKey'
-            required: true
+            output: true
             description: |
               A PublicKey describes a public key.
             properties:
             - !ruby/object:Api::Type::String
               name: 'key'
+              output: true
               description: |
                 Required. A public key. When this is specified in a request, the padding and encoding can be any of the options described by the respective 'KeyType' value. When this is generated by the service, it will always be an RFC 5280 SubjectPublicKeyInfo structure containing an algorithm identifier and a key. A base64-encoded string.
             - !ruby/object:Api::Type::Enum
-              name: 'type'
-              required: true
+              name: 'format'
+              output: true
               description: |
-                Types of public keys that are supported. At a minimum, we support RSA and ECDSA, for the key sizes or curves listed: https://cloud.google.com/kms/docs/algorithms#asymmetric_signing_algorithms
+                Types of public keys that are supported. At a minimum, we support RSA, for the key sizes or curves listed: https://cloud.google.com/kms/docs/algorithms#asymmetric_signing_algorithms
```

Comment:
perhaps a more accurate description is "the key formats that are supported".

also, FWIW, we still support ECDSA keys. do you know if the "ECDSA" part was dropped from the API or here?

(and same for publicKey.format below)

Reply:
It doesn't look present in the REST API docs, which is why I removed it here